code/modules/research/techweb/all_nodes.dm

```diff
@@ -539,6 +539,15 @@
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
 	export_price = 5000
 
+/datum/techweb_node/rld
+	id = "rld"
+	display_name = "Advanced Sanitation Technology"
+	description = "Clean things better, faster, stronger, and harder!"
+	prereq_ids = list("janitor")
+	design_ids = list("rld")
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 1000)
+	export_price = 5000
```

Comment:
Export price is typically 2x the cost, so 2000 here

Reply:
ok fixing that

Reply:
ok fixed that